toucan_connectors/facebook_ads/facebook_ads_connector.py

```diff
@@ -0,0 +1,141 @@
+import os
+from pathlib import Path
+from typing import Dict, List, Optional
+from urllib.parse import urljoin
+
+import pandas as pd
+import requests
+from pydantic import Field
+
+from toucan_connectors.oauth2_connector.oauth2connector import (
+    OAuth2Connector,
+    OAuth2ConnectorConfig,
+)
+from toucan_connectors.toucan_connector import (
+    ConnectorSecretsForm,
+    ToucanConnector,
+    ToucanDataSource,
+)
+
+from .helpers import FacebookadsDataKind, has_next_page
+
+API_BASE_ROUTE = 'https://graph.facebook.com/v10.0/'
+
+API_ENDPOINTS_MAPPING = {
+    FacebookadsDataKind.campaigns: 'act_{act_id}/campaigns',
+    FacebookadsDataKind.ads_under_campaign: '{campaign_id}/ads',
+    FacebookadsDataKind.all_ads: 'act_{act_id}/ads',
+}
+
+AUTHORIZATION_URL = 'https://www.facebook.com/v10.0/dialog/oauth'
+SCOPES = ''
```

Comment:
We don't have to set any scope ?

Reply:
Fixé ici : https://github.com/ToucanToco/toucan-connectors/pull/349/commits/76902190a2539f21bceb5cfd7edb0c965c189fd4